### ISSUE TITLE
This package has a devDependencies on an old rollup version (3) that conflicts with the one (4) used by recent vite

### ISSUE THREAD

**Issue by stephanemagnenat** [illright/unplugin-lezer#6]
This leads to errors when using this package with some other packages that have a more recent (4.x) rollup dependency:
```
 WARN  Issues with peer dependencies found
.
└─┬ unplugin-lezer 1.0.0
  └── ✕ unmet peer rollup@^3: found 4.18.1
```

In my case the source of the newer rollup version is vite itself (`pnpm why rollup`):
```
...
vite 5.3.4
└── rollup 4.18.1
...
```

Are the devDependencies on all build systems (rollup, vite, webpack, etc.) really necessary? Could there be a switch to choose which one to depend on?

**Comment by illright** [comment]
I released 1.0.1 that should fix it, could you check please?

**Comment by stephanemagnenat** [comment]
It looks like it solves the problem, thank you very much!
